openms-wiki/contributors-quickstart-guide.md

```diff
@@ -0,0 +1,64 @@
+Contribute to OpenMS
+====================
+
+If you would like to contribute to OpenMS:
+
+* Familiarise yourself with our [online documentation](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/release/latest/html/index.html).
+
+* Learn how to [build OpenMS](build-openms-from-source.md).
+
+* Check out the [OpenMS tutorial for developers](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/release/latest/html/OpenMS_tutorial.html).
+
+Any questions can be directed at the mailing list.
```

Comment:
For any questions, please [contact us](contact-us.md] at [open-ms-general mailing list]().

Reply:
done